ceph-salt-formula/salt/ceph-salt/apply/cephconfigure.sls

```diff
@@ -2,14 +2,10 @@
 
 {% if 'cephadm' in grains['ceph-salt']['roles'] or 'admin' in grains['ceph-salt']['roles'] %}
 
-{{ macros.begin_stage('Find an admin host') }}
-
```

Comment:
What is the rationale for dropping the "Find an admin host" stage begin and end messages?

Reply:
It's not being dropped.. it was moved to `ceph_orch.set_admin_host` state [1] because we don't want to show that message if a reboot is not needed [2].

Now you may be asking "why not using a Jinja `if` condition" for that?
Answer is... because `ceph-salt:execution:reboot_needed` is set dinamically in [3] and state file is renderer when you start `state.apply` (and will not be renderer again even if you change the value of grains used in `if` conditions during runtime).


[1] https://github.com/ceph/ceph-salt/pull/345/files/88380c09fee8dd750f8d4c20b183d4d9a9cb901f#diff-dbf4c37a5be102b76d21bbb73d262c20R16

[2] https://github.com/ceph/ceph-salt/pull/345/files/88380c09fee8dd750f8d4c20b183d4d9a9cb901f#diff-1a2c1f3cc1d8d46ea1184ada6d3a448bR35

[3] https://github.com/ceph/ceph-salt/pull/345/files/88380c09fee8dd750f8d4c20b183d4d9a9cb901f#diff-1a2c1f3cc1d8d46ea1184ada6d3a448bR25